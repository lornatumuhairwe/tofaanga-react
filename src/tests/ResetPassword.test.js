import React from 'react';
import renderer from 'react-test-renderer';
import ResetPasswordForm from '../components/Authentication/ResetPassword';

describe('LoginForm snapshot', () => {
  it('Login Form renders snapshot', () => {
    const Resetpasswordform = renderer.create(<ResetPasswordForm />);
    expect(Resetpasswordform.toJSON()).toMatchSnapshot();
  });
});
