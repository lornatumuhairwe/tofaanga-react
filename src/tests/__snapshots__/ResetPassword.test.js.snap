// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm snapshot Login Form renders snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm-6 col-md-4 col-md-offset-4"
    >
      <h1
        className="text-center login-title"
      >
        Reset your password
      </h1>
      <div
        className="notifications-wrapper"
        style={Object {}}
      />
      <div
        className="account-wall"
      >
        <img
          alt=""
          className="profile-img"
          src="https://lh5.googleusercontent.com/-b0-k99FZlyE/AAAAAAAAAAI/AAAAAAAAAAA/eu7opA4byxI/photo.jpg?sz=120"
        />
        <form
          className="form-signin"
          onSubmit={[Function]}
        >
          <input
            autoFocus={true}
            className="form-control"
            id="email"
            onChange={[Function]}
            placeholder="Email"
            required={true}
            type="text"
            value=""
          />
          <input
            className="form-control"
            id="pwd"
            onChange={[Function]}
            placeholder="Password"
            required={true}
            type="password"
            value=""
          />
          <input
            className="form-control"
            id="pwd"
            onChange={[Function]}
            placeholder="Confirm Password"
            required={true}
            type="password"
            value=""
          />
          <button
            className="btn btn-lg btn-primary btn-block btn btn-default"
            disabled={false}
            type="submit"
          >
            Reset Password
          </button>
        </form>
      </div>
      <a
        className="text-center new-account"
        href=""
        onClick={undefined}
      >
        Create an account 
      </a>
    </div>
  </div>
</div>
`;
